Index: trunk/configure.in
===================================================================
--- trunnk/configure.in	(revision 682)
+++ trunnk/configure.in	(revision 683)
@@ -1243,14 +1243,8 @@
 
 dnl --- Groupwise flags
 if test "x${with_soup}" != "xno"; then
-PKG_CHECK_MODULES(SOUPTEMP, libsoup-2.4, have_libsoup_24=yes, have_libsoup_24=no)
-if test $have_libsoup_24 = yes; then
-    LIBSOUP_REQUIRED=2.3.0
-    LIBSOUP=libsoup-2.4
-else
-    LIBSOUP_REQUIRED=2.2.2
-    LIBSOUP=libsoup-2.2
-fi
+LIBSOUP_REQUIRED=2.2.2
+LIBSOUP=libsoup-2.2
 AC_SUBST(LIBSOUP_REQUIRED)
 AC_SUBST(LIBSOUP)
 EVO_SET_COMPILE_FLAGS(SOUP, $LIBSOUP >= $LIBSOUP_REQUIRED)
