Index: e2fsprogs-1.38/lib/ext2fs/ismounted.c
===================================================================
--- e2fsprogs-1.38.orig/lib/ext2fs/ismounted.c	2009-07-28 13:48:34.000000000 +0200
+++ e2fsprogs-1.38/lib/ext2fs/ismounted.c	2009-07-28 13:49:10.000000000 +0200
@@ -147,7 +147,7 @@
 is_root:
 #define TEST_FILE "/.ismount-test-file"		
 		*mount_flags |= EXT2_MF_ISROOT;
-		fd = open(TEST_FILE, O_RDWR|O_CREAT);
+		fd = open(TEST_FILE, O_RDWR|O_CREAT, 0600);
 		if (fd < 0) {
 			if (errno == EROFS)
 				*mount_flags |= EXT2_MF_READONLY;
