---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- matchbox-keyboard.orig/configure.ac
+++ matchbox-keyboard/configure.ac
@@ -49,11 +49,11 @@ AC_ARG_ENABLE(applet,
    		enable_applet=$enableval, 
 		enable_applet=no)	    
 AM_CONDITIONAL(WANT_APPLET, test x$enable_applet = xyes)
 
 AC_ARG_ENABLE(debug,
-  AC_HELP_STRING([--enable-debug], [enable debug (verbose) build]),,
+  AC_HELP_STRING([--enable-debug], [enable debug (verbose) build]),
      enable_debug=$enableval, enable_debug=no )
 
 AC_ARG_WITH(expat-includes,    
   AC_HELP_STRING([--with-expat-includes=DIR], [Use Expat includes in DIR]),
 	   expat_includes=$withval, expat_includes=yes)
